projects/templates/plop-templates/README.md.hbs

```diff
@@ -1,30 +1,68 @@
-## Description
+## Overview
 
 
 
 ### Usage
 
+
 [![See it on NPM!](https://img.shields.io/npm/v/@spectrum-web-components/{{ name }}?style=for-the-badge)](https://www.npmjs.com/package/@spectrum-web-components/{{ name }})
 [![How big is this package in your project?](https://img.shields.io/bundlephobia/minzip/@spectrum-web-components/{{ name }}?style=for-the-badge)](https://bundlephobia.com/result?p=@spectrum-web-components/{{ name }})
 
-```
+```bash
 yarn add @spectrum-web-components/{{ name }}
 ```
 
 Import the side effectful registration of `<{{> tagnamePartial}}>` via:
 
-```
+```ts
 import '@spectrum-web-components/{{ name }}/{{> tagnamePartial}}.js';
 ```
 
 When looking to leverage the `{{className name}}` base class as a type and/or for extension purposes, do so via:
 
-```
+```ts
 import { {{className name}} } from '@spectrum-web-components/{{ name }}';
 ```
 
-## Example
+## Anatomy
+
+<--
+Information and examples about the parts of the component
+(eg., labels, icons, etc.)
+of the component. -->
 
 ```html
 <{{> tagnamePartial}}></{{> tagnamePartial}}>
-```
\ No newline at end of file
+```
+
+### Options
```

Comment:
Updated the new component readme template to match the recommended heading structure.